reboot-via-dsme.sh

```diff
@@ -0,0 +1,15 @@
+# Define shell functions for interactive shells only
+[ -z "$PS1" ] && return
```

Comment:
See [How to check if a shell is login/interactive/batch](http://unix.stackexchange.com/questions/26676/how-to-check-if-a-shell-is-login-interactive-batch) for some other approaches. `[ -z "$PS1" ]` might not always indicate interactive shell.